2019-10-11 16:23:36,039 [28] ERROR NotaCreditoController  - Error: System.ArgumentNullException: El valor no puede ser nulo.
Nombre del parámetro: source
   en System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   en FasceMVC.Controllers.NotaCreditoController.GuardarDatosEditados(Detalle detalle) en D:\FST\VS2017\MVC\FACSE\FasceMVC\Controllers\NotaCreditoController.cs:línea 211
2019-10-11 16:24:01,886 [32] ERROR NotaCreditoController  - Error: System.ArgumentNullException: El valor no puede ser nulo.
Nombre del parámetro: source
   en System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   en FasceMVC.Controllers.NotaCreditoController.GuardarDatosEditados(Detalle detalle) en D:\FST\VS2017\MVC\FACSE\FasceMVC\Controllers\NotaCreditoController.cs:línea 211
2019-10-11 16:52:52,974 [47] ERROR NotaCreditoController  - Error: System.Data.Entity.Core.EntityException: The underlying provider failed on Open. ---> System.Data.SqlClient.SqlException: Valor de tiempo de espera caducado. El periodo de tiempo de espera se agotó al intentar consumir la confirmación del protocolo de enlace previo al inicio de sesión. El motivo podría ser que se produjo un error en el protocolo de enlace o que el servidor no pudo responder a tiempo.  El tiempo invertido al intentar conectarse a este servidor fue: [Previo al inicio de sesión] inicialización=13525; protocolo de enlace=2578;  ---> System.ComponentModel.Win32Exception: Tiempo de espera de la operación de espera agotado
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.Open()
   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.<Open>b__36(DbConnection t, DbConnectionInterceptionContext c)
   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   en System.Data.Entity.Core.EntityClient.EntityConnection.<Open>b__2()
   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute(Action operation)
   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en Metodos.ClsNotaCredito.ConsultarJsonFactura(Guid idEmisor, String prefijo, Decimal numeroFactura) en D:\FST\VS2017\MVC\FACSE\Metodos\ClsNotaCredito.cs:línea 48
   en FasceMVC.Controllers.EditarFacturaController.Index(NotaCredito model) en D:\FST\VS2017\MVC\FACSE\FasceMVC\Controllers\EditarFacturaController.cs:línea 59
